data/core/units/orcs/Adept.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Adept
+    name= _ "Orcish Adept"
+    race=orc
+    image="units/orcs/shaman.png"
+    profile="portraits/orcs/adept.webp"
+    hitpoints=24
+    movement_type=orcishfoot
+    [defense]
+        village=30
+    [/defense]
+    movement=5
+    experience=50
+    level=1
+    alignment=chaotic
+    advances_to=Orcish Shaman
+    cost=21
```

Comment:
Cost too high. Might be like 16 or 17.

Reply:
Good joke. 9.

Reply:
So you're requesting I set the cost to 9?

Reply:
9 seems more appropriate for a level 0 unit, but this is level 1?

Reply:
reduced to 16 in 28ef66c, but then increased back to 20 in eec265c?